packages/salesforcedx-vscode-soql/package.json

```diff
@@ -118,6 +118,13 @@
     ],
     "packageUpdates": {
       "main": "dist/index.js",
+      "serverPath": [
+        "node_modules",
+        "@salesforce",
+        "soql-language-server",
+        "lib",
+        "server.js"
+      ],
```

Comment:
This updates where we get the server file from when running as a vsix.  You can see the dev version in line 100 of this file.